api/handler/get.go

```diff
@@ -73,7 +73,7 @@ func writeHeaders(h http.Header, info *layer.ObjectInfo) {
 	h.Set(api.ETag, info.HashSum)
 
 	for key, val := range info.Headers {
-		h.Set(api.MetadataPrefix+key, val)
+		h[api.MetadataPrefix+key] = []string{val}
```

Comment:
`api.MetadataPrefix+key` should not be canonized? Maybe it worth extra comment in code?

Edit 1: Okay, too late, it is merged already :smile:

Reply:
> `api.MetadataPrefix+key` should not be canonized? Maybe it worth extra comment in code?
> 
> Edit 1: Okay, too late, it is merged already

[Amazon S3 stores user-defined metadata keys in lowercase.](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingMetadata.html#UserMetadata)